client/dashboard/sites/features.ts

```diff
@@ -117,3 +117,20 @@ export function canTransferSite( site: Site, user: User ) {
 	const isSiteOwner = site.site_owner === user.ID;
 	return isAllowedSiteType && isSiteOwner;
 }
+
+export function canLeaveSite( site: Site ) {
+	return ! site.is_wpcom_staging_site;
+}
+
+export function canResetSite( site: Site ) {
+	return ! site.is_wpcom_staging_site;
+}
+
+export function canDeleteSite( site: Site, isStagingRedesignEnabled?: boolean ) {
```

Comment:
I think it's cleaner if we get `isStagingRedesignEnabled` here from config instead of being passed from the caller. This way, when we do cleanup later, we can only delete it here. What do you think?